code/modules/mob/living/carbon/human/body_accessories.dm

```diff
@@ -121,4 +122,85 @@ GLOBAL_LIST_INIT(body_accessory_by_species, list("None" = null))
 	name = "Bee Tail"
 	icon_state = "wryntail"
 	allowed_species = list("Wryn")
-	
+
+//Moth
+//Wings
+/datum/body_accessory/tail/moth
+	icon = 'icons/mob/sprite_accessories/moth/moth_wings.dmi'
+	animated_icon = null
+	name = "Plain Wings"
+	icon_state = "plain"
+	allowed_species = list("Moth")
+	is_wing = TRUE
```

Comment:
if `is_wing` is only true here, wouldn't it be better just to do an `istype` check?

Reply:
I'm unsure, var lookup is slightly more performant, and modular for admin shenanigans. Istype is neater overall.

Reply:
you could make this a helper, something like `is_wing()` 

I don't like this being a var when its used twice and it could just be an istype.